numfmt_test.go

```diff
@@ -3613,3 +3613,74 @@ func TestNumFmt(t *testing.T) {
 	assert.Equal(t, ErrUnsupportedNumberFormat, err)
 	assert.False(t, changeNumFmtCode)
 }
+
+func TestLangNumFmtFunc(t *testing.T) {
```

Comment:
Did you test each built-in number format with different data time pattern settings on real Office application?

Reply:
So, what i should do is:
1. Place the numfmt.xlsx file inside the ./test directory.
2. In the tests, set the langNumFmt for the cells in sequence, with each column corresponding to a different region.
3. Retrieve the cell values and compare them to ensure they are correct.

It's not just about the new additions i have made, but also about including zh-cn and en-us as well,right?
In addition to langNumFmt, I also need to do the part of Built-in number format code, right?

Reply:
You needn't add new workbook to the test directory, the workbook just for test, open the workbook in the real spreadsheet apps to observe formats result in different data time pattern settings, and using the formatted result as expected result in the unit test.